maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -481,14 +503,29 @@ def step(self, tick: int):
     def post_step(self, tick: int):
         # Update energy to the environment metrices.
         total_energy: float = 0.0
+        total_energy_cost: float = 0.0
         for pm in self._machines:
             if pm.oversubscribable and pm.cpu_cores_allocated > pm.cpu_cores_capacity:
                 self._total_oversubscriptions += 1
             total_energy += pm.energy_consumption
+            # Update the energy consumption cost.
+            pm_cost = (
+                pm.energy_consumption
```

Comment:
change the unit to kwh, remove _w_per_kw.

The corresponding comment I added before is still unsolved